packages/itwin/grouping-mapping-widget/src/components/Mappings/Extraction/ExtractionStatus.tsx

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useEffect, useState } from "react";
+import * as React from "react";
+import { STATUS_CHECK_INTERVAL } from "../../Constants";
+import { StartingExtractionState } from "./ExtractionStates/StartingExtractionState";
+import { FailedExtractionState } from "./ExtractionStates/FailedExtractionState";
+import { QueuedExtractionState } from "./ExtractionStates/QueuedExtractionState";
+import { SucceededExtractionState } from "./ExtractionStates/SucceededExtractionState";
+import { RunningExtractionState } from "./ExtractionStates/RunningExtractionState";
+import "./ExtractionStatus.scss";
+
+export enum ExtractionStates {
+  None,
+  Starting,
+  Queued,
+  Running,
+  Succeeded,
+  Failed,
+}
+
+interface ExtractionStatusProps {
+  state: ExtractionStates | undefined;
+  clearExtractionState: () => void;
+}
+
+export const ExtractionStatus = ({
+  state,
+  clearExtractionState,
+}: ExtractionStatusProps) => {
+
```

Comment:
Don't add empty lines at the beginning or the end of a block of code.

Reply:
I have made this change.